core/src/main/scala/org/apache/spark/util/logging/DriverLogger.scala

```diff
@@ -66,7 +66,7 @@ private[spark] class DriverLogger(conf: SparkConf) extends Logging {
       // `AbstractFilterable.Builder.asBuilder()` method will return `Any` in Scala.
       val builder: Log4jFileAppender.Builder[_] = Log4jFileAppender.newBuilder()
       builder.withAppend(false)
-      builder.withBufferedIo(false)
+      builder.setBufferedIo(false)
```

Comment:
https://github.com/apache/logging-log4j2/blob/55bbf911270f1ba6a13f6d3b85f1f83e9818a53d/log4j-core/src/main/java/org/apache/logging/log4j/core/appender/AbstractOutputStreamAppender.java#L69-L89

<img width="830" alt="image" src="https://user-images.githubusercontent.com/1475305/178988548-e5537649-ccbf-4a4b-b836-1dc86ef043de.png">

just use `setBufferedIo` instead of `withBufferedIo`

Reply:
Only this one case